src/lib/isNavigationEnabled.js

```diff
@@ -0,0 +1,10 @@
+import 'url-search-params-polyfill'
+
+const isNavigationEnabled = (search = '') => {
```

Comment:
Maybe we could merge this file with `lib/getFilteredAppsFromSearch` into a `helpers/navigation`. What do you think ?

Reply:
`getFilteredAppsFromSearch` is not related to navigation, and it seems clearer to have these helpers in dedicated file since they only do one thing imho

Reply:
I `getFilteredAppsFromSearch` is actually filtering application based on what is present in the query string, I don't see how we could implement something more related to navigation.